CMakeLists.txt

```diff
@@ -228,6 +230,21 @@ if(APPLE)
     MACOSX_PACKAGE_LOCATION Resources)
 endif(APPLE)
 
+set(MSGPACK_CPP ${CMAKE_CURRENT_BINARY_DIR}/messages)
```

Comment:
This a hack to force CMake to generate this file, even if it doesn't think it will change. The other way (now that I think about it, slightly more elegant) would be to list all Python files that can affect generated file as dependencies of messages.h, and make sure that this list is up to date at all times.